uelc/main/views.py

```diff
@@ -212,6 +206,7 @@ def notify_facilitators(self, request, path, notification):
         try:
             socket.recv(zmq.NOBLOCK)
         except zmq.ZMQError:
+            socket.recv()
```

Comment:
Nik, I am not sure what this will do to Behave. I was noticing that the socket was not receiving the notification that a user landed on a gateblock so I put this in the except and it fixed the issue.